build/test.ps1

```diff
@@ -43,16 +43,22 @@ if ($Env:ENABLE_QIRRUNTIME -ne "false") {
     if ($LastExitCode -ne 0) {
         $script:all_ok = $False
     }
+
     $qirTests = (Join-Path $PSScriptRoot "../src/Qir/Tests")
     & "$qirTests/test-qir-tests.ps1"
     if ($LastExitCode -ne 0) {
         $script:all_ok = $False
     }
+
     $qirSamples = (Join-Path $PSScriptRoot "../src/Qir/Samples")
     & "$qirSamples/test-qir-samples.ps1"
     if ($LastExitCode -ne 0) {
         $script:all_ok = $False
     }
+
+    if ($LastExitCode -ne 0) {
+        $script:all_ok = $False
+    }
```

Comment:
FYI: I see a sequence of 2 identical `if`s:
```ps1
if ($LastExitCode -ne 0) { ..
if ($LastExitCode -ne 0) { ..
```

Reply:
I removed the second one. This had a similar cause that the one in your previous [comment](https://github.com/microsoft/qsharp-runtime/pull/761#discussion_r661687131). The second one was used to check for the result of controller tests.